WalletWasabi.Fluent/Views/Wallets/Home/History/Columns/IndicatorsColumnView.axaml

```diff
@@ -58,6 +58,9 @@
 
       <PathIcon Opacity="0.6"
                 Classes.NormalTransaction="{Binding IsNormalTransactionDisplayed}"
```

Comment:
This should be removed along with the `IsNormalTransactionDisplayed` property.
```suggestion
```

Reply:
All changes have been updated.